src/cloud/live/sens-data-over.md

```diff
@@ -163,6 +163,18 @@ This process **overwrites** the store configuration; only do the following if th
 
 1. Add, commit, and push code changes to update a remote environment.
 
+## Configuration Hierarchy
+
+With Cloud Commerce there are several ways to set admin configurations and it is import to understand the order of priority, if a setting is added in two or more ways, the highest priority method takes precedence.
+
+The current hierarchy is as follows, the highest priority is top of the list.
+
+-  Cloud Admin UI
+-  .magento.app.yaml
+-  env.php
+-  config.php
+-  Database
```

Comment:
@BarnyShergold  I'm not sure about this section.  I think some of this information is conveyed in the [Configuration logic flow](https://devdocs.magento.com/cloud/live/sens-data-over.html#configuration-logic-flow) section. That seems like the place to add any new information. 

So for Cloud -- We recommend configuring through Magento Admin UI, .magento.app.yaml, and using environment variables.   We don't recommend manually editing env.php, config.php, or the database.    I know that there are ways to edit the files manually, and I understand the need to say how the values in different locations are applied.  I'm thinking that enumerating each location in a table with the location, followed by description of how the values are added in each one. 
 Not exactly sure.  What do you think?